File: Neon.Server/Domain/Models/Asset.cs
using System;

namespace Neon.Server.Models {
    public class Asset {
        public string Id { get; set;}
        public string Name { get; set; }
        public int DisplayTime { get; set; }
        public bool IsActive { get; set; }
        public int Order { get; set; }
        public EAssetType Type { get; set; }
        public DateTime? NotBefore { get; set; }
        public DateTime? NotAfter { get; set; }
    }
}

File: Neon.Client/src/models/Asset.ts
export default class Asset {
    public id: string;
    public name: string;
    public type: EAssetType;
    public isActive: boolean;
    public order: number;
    public displayTime: number;
    public notBefore: string;
    public notAfter: string;
}

export enum EAssetType {
    IMAGE,
    HTML
}